FundacionParaguaya/MentorApp
# Families screen

**Describe the changes**
Implementation of the families screen

**To Test**
Login with user: t/demo pass: demo to access the testing env.
Go to families screen. See the list, see that is updates if you are online every time you visit the screen. See that it does not update if you are offline. See that clicking on a family takes you to the overview. There is no continue button. You can only click on indicators that have either a priority or an achievement. You can view the priority or achievement. The information is read only. The save button on the priority and achievement screen is hidden.

To test it actually works take a survey, create some achievements and priorities, see that the data is available through the families screen.

It also includes now error message - #359
